### DISCUSSION TITLE
Add information where to create new repositories in DataCite Fabrica

### DISCUSSION THREAD

**Discussion by paulvierkant** [datacite/datacite-suggestions#131] [General Suggestions]
### What is the problem that your suggestion solves?

Under the "Repositories" tab in Fabrica, it says "New repositories can't be created from this page." However, as a user who is new to Fabrica, I would not know where to create a repository.

### What solution might meet your needs?

Add information or a link where to find the right place for creating new repositories.

### Your name

Paul Vierkant

### Your organization

DataCite

### What group(s) would benefit from your suggestion?

- [X] DataCite members registering DOIs and metadata and administering accounts
- [ ] Developers building integrations with DataCite services
- [ ] Harvesters who index DataCite DOIs and metadata
- [ ] Individuals or institutions analyzing DataCite DOIs and metadata
- [X] DataCite staff
- [ ] Other

**Comment by codycooperross** [comment]
Hi @paulvierkant ! We updated the alert here to help clarify where to add a repository, which we hope will be a helpful change for Consortia managing repositories. Feel free to take a look and let us know if you have any other feedback.
